core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -58,7 +59,10 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
     super.setUp(testInfo)
-    consumer = TestUtils.createConsumer(TestUtils.getBrokerListStrFromServers(servers), securityProtocol = SecurityProtocol.PLAINTEXT)
+    consumer = TestUtils.createConsumer(bootstrapServers(
+      listenerName = ListenerName.forSecurityProtocol(SecurityProtocol.PLAINTEXT)),
+      securityProtocol = SecurityProtocol.PLAINTEXT
+    )
```

Comment:
I think the formatting is incorrect here. Should it be?

```suggestion
    consumer = TestUtils.createConsumer(
      bootstrapServers(ListenerName.forSecurityProtocol(SecurityProtocol.PLAINTEXT)),
      securityProtocol = SecurityProtocol.PLAINTEXT
    )
```